src/components/shared/TableRow.jsx

```diff
@@ -1,18 +1,67 @@
-import React from 'react';
+/* eslint react/prop-types: 0 */
```

Comment:
We can disable this globally in `.eslintrc` so you don't have to disable per file - if we're running into cases when it seems this is common we should make the rule work for us

Reply:
(non-blocking)

Reply:
Set to     "react/prop-types": 1